TrafficCapture/KafkaPrinter/src/main/java/org/opensearch/KafkaPrinter.java

```diff
@@ -0,0 +1,55 @@
+package org.opensearch;
+
+
+import java.util.ArrayList;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.errors.WakeupException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Properties;
+
+public class KafkaPrinter {
+    private static final Logger log = LoggerFactory.getLogger(KafkaPrinter.class);
+
+    public static void main(String[] args) {
+        String bootstrapServers = args[1];
+        String groupId = "GroupID";
```

Comment:
Could we name this something like `default-logging-group`? I know we aren't using right now as we only have a single consumer so not a big deal, but in future I imagine we may pass this in